Various GUI updates

alustig3 opened the pull request:
### Custom Variable Dialog
- fixed a bug where the slider control could potentially expand to take up too much space
- added a "separator" control that allows you to add horizontal line separators within a control tab.
<img width="928" alt="gui_editor_with_separators" src="https://user-images.githubusercontent.com/8128628/189495965-b675ce38-0d15-446a-93db-d54cbb2ac4d0.png">
<img width="374" alt="variable_dialog_separator_demo" src="https://user-images.githubusercontent.com/8128628/189495972-e15894dd-b2ea-4a57-902e-9905ecf6a150.png">

### Experiment Configuration
- added icons and adjusted layout slightly
<img width="912" alt="config_exp_before" src="https://user-images.githubusercontent.com/8128628/189496033-7e657552-1248-419e-a033-56e06136fea0.png">
<img width="904" alt="config_exp_after" src="https://user-images.githubusercontent.com/8128628/189496043-5c190bc6-d38d-4958-a862-d23547b58068.png">

### Setups
- The "Configure Selected" buttons are now only enabled if there is at least one setup selected

### Other
- fixed typo in log message

ThomasAkam commented:
Thanks Andy, looks great.

The only change I'm not sure about is the new stretch behaviour of the experiment_select QComboBox.  I think I preferred how it looked when the QComboBox stretched rather than having it fixed width with an empty space beside it that stretches.  Unless you feel strongly about it I'd be inclined to stick with the old behaviour there.

alustig3 commented:
I removed the stretch behavior you referred to.

I also updated the pause plotting button. It now has an icon and the button text updates to show say "Resume plotting" if it is paused, and "Pause plotting" if it is running.

ThomasAkam commented:
Thanks, I'll merge the pull request now.
